hack/run-wmcb-ci-e2e-test.sh

```diff
@@ -57,6 +57,11 @@ fi
 sed -i "s~ARTIFACT_DIR_VALUE~${ARTIFACT_DIR}~g" internal/test/wmcb/deploy/job.yaml
 sed -i "s~REPLACE_IMAGE~${WMCB_IMAGE}~g" internal/test/wmcb/deploy/job.yaml
 
+# If DOCKER_RUNTIME is set, set that value for flag -dockerRuntime in job.yaml.
+# If it is unset, default value taken will be true.
+DOCKER_RUNTIME=${DOCKER_RUNTIME:-true}
```

Comment:
For my own learning, why is the DOCKER_RUNTIME env var being taken from the release repo AWS e2e CCM config specifically?

Reply:
@alinaryan We are passing dockerRuntime=false from CCM job so we can test the installation and setting of containerd as the runtime through kubelet arguments using only CCM job.